src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -144,6 +153,30 @@ export class ClrDatagridColumn<T = any> extends DatagridFilterRegistrar<T, Datag
     this._sortSubscription.unsubscribe();
   }
 
+  /*
+     * What type is this column?  This defaults to STRING, but can also be
+     * set to NUMBER.  Unsupported types default to STRING. Users can set it
+     * via the [clrDgColType] input by setting it to 'string' or 'number'.
+     */
+
+  private _colType: ClrDatagridColumnType = ClrDatagridColumnType.STRING;
+  public get colType() {
+    if (this._colType === ClrDatagridColumnType.NUMBER) {
+      return 'number';
```

Comment:
No offense, but this looks a bit silly to me. You receive a string as an input, convert it to an enum, and then you re-convert it to a string when you use it in your own template? Either just use strings everywhere, or embrace enums and don't convert back to strings.

To be clear, I think the best solution is simply to use a string union type rather than an enum:
```
@Input('clrDgColType') colType: "string" | "number" = "string";
```
No getter or setter needed, and the default works fine. If the consumer gives us anything other than these two strings, AoT compilation would fail with a clear message.